src/database/search_handler.cc

```diff
@@ -677,7 +677,7 @@ std::string SortParser::parse(std::string& addColumns, std::string& addJoin)
         } else {
             log_warning("Unknown sort direction '{}' in '{}'", seg, sortCrit);
         }
-        if (!colMapper || !colMapper->mapQuotedList(sort, seg, (desc ? "DESC" : "ASC"))) {
+        if (!colMapper->mapQuotedList(sort, seg, (desc ? "DESC" : "ASC"))) {
```

Comment:
can you add a check for `colMapper` and `metaMapper` in the begin of the method
```
if (sortCrit.empty() || !colMapper || !metaMapper)
```

Reply:
Fixed.